### PR TITLE
Added solar thermal to final demand for space heating and hot water q…

### PR DESCRIPTION
**Pull request by ChaelKruip**
…ueries

### CODE REVIEW

**Comment by ChaelKruip** [comment]
@antw I was hoping that we could also put this on the production server actually (after reviewing). One of our clients needs it.

File: gqueries/output_elements/output_series/bezier_56_heat_production_in_households/solar_used_for_heating_in_households.gql [hunk 1]
```diff
@@ -4,5 +4,6 @@
 - query =
     SUM(
       V(CHILDREN(V(households_useful_demand_for_space_heating_after_insulation_and_solar_heater)),input_of_solar_radiation),
-      V(CHILDREN(V(households_useful_demand_for_space_heating_after_insulation_and_solar_heater)),input_of_solar_thermal)      
+      V(CHILDREN(V(households_useful_demand_for_space_heating_after_insulation_and_solar_heater)),input_of_solar_thermal),
```
Comment (antw) [inline gqueries/output_elements/output_series/bezier_56_heat_production_in_households/solar_used_for_heating_in_households.gql]: None of the child nodes (input nodes) have a solar radiation or solar thermal carrier input, and so will always return zero. Is this intentional?

```ruby
CHILDREN(V(households_useful_demand_for_space_heating_after_insulation_and_solar_heater))
  .select { |conv| conv.input(:solar_thermal) }

# => [ <empty> ]

CHILDREN(V(households_useful_demand_for_space_heating_after_insulation_and_solar_heater))
  .select { |conv| conv.input(:solar_radiation) }

# => [ <empty> ]

File: gqueries/output_elements/output_series/bezier_65_source_of_hot_water_in_households/solar_used_for_hot_water_in_households.gql [hunk 2]
```diff
@@ -4,5 +4,6 @@
 - query =
     SUM(
       V(CHILDREN(V(households_useful_demand_for_hot_water_after_solar_heater_and_p2h)),input_of_solar_radiation),
-      V(CHILDREN(V(households_useful_demand_for_hot_water_after_solar_heater_and_p2h)),input_of_solar_thermal)      
+      V(CHILDREN(V(households_useful_demand_for_hot_water_after_solar_heater_and_p2h)),input_of_solar_thermal),
```
Comment (antw) [inline gqueries/output_elements/output_series/bezier_65_source_of_hot_water_in_households/solar_used_for_hot_water_in_households.gql]: Same as above, this node's children don't have any solar thermal or solar radiation inputs. Am I misunderstanding what these lines do? 😆 

```ruby
CHILDREN(V(households_useful_demand_for_hot_water_after_solar_heater_and_p2h))
  .select { |conv| conv.input(:solar_thermal) }

# => [ <empty> ]

CHILDREN(V(households_useful_demand_for_hot_water_after_solar_heater_and_p2h))
  .select { |conv| conv.input(:solar_radiation) }

# => [ <empty> ]
```

Comment (ChaelKruip) [inline gqueries/output_elements/output_series/bezier_65_source_of_hot_water_in_households/solar_used_for_hot_water_in_households.gql]: No, you get it 200% correctly. These lines are doing nothing at the moment. They also do no real harm (except confusing you 😉 ).

**Comment by antw** [comment]
> I was hoping that we could also put this on the production server actually (after reviewing).

Done!

| Before | After |
| :---: | :---: |
| screen shot 2017-01-05 at 14 52 46 | screen shot 2017-01-05 at 14 53 45 |

**Pull request merged**